README.md

```diff
@@ -103,10 +105,11 @@ Some of the most important changes for each update are as follows:
 | v1.8.0    | 29/08/2023   | New End Screen Menu, animal behaviour improvements, wider variety of planet textures |
 | v1.9.0    | 12/09/2023   | Boost speed increased, on planet UI, new planet scanning info |
 | v1.10.0   | 10/10/2023   | Expanded planet visuals, correct visuals in planet exploration, music and sound effects, new in planet models, improved planet info panel |
+| v2.0.0    | 24/10/2023   | More threatening black hole, new names for planets and animals, new and improved info panels  |
```

Comment:
I think it would be professional to say "information" panels, and try to specify that it's on the planet to let our adoring fans know more!

Reply:
I changed it to "information" however I left it non specific due to both the planet and space panels being updated in this build